### PR TITLE
[RTE] Resolve issue with table size selection on mobile

### PR DESCRIPTION
**Pull request by palatter**
# What

Fixes table size selection view on **mobile**.

Before the fix:

 
image

After the fix: 

image

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by JoshuaLai — APPROVED** [review]

File: packages/react-components/src/components/styles/RichTextEditor.styles.ts [hunk 1]
```diff
@@ -288,12 +285,9 @@ export const insertTableMenuTablePane = mergeStyles({
  */
 export const insertTableMenuFocusZone = (theme: Theme): string => {
   return mergeStyles({
-    lineHeight: '12px',
-    // fixed width is required to show columns in a grid correctly
-    width: '5rem',
```
Comment (vhuseinova-msft) [inline packages/react-components/src/components/styles/RichTextEditor.styles.ts:293]: Just wanted  to confirm that the changes were tested in sample app and storybook

**Comment by vhuseinova-msft** [comment]
@palatter Do you know why the "table cells" doesn't look square on mobile?
image

**Review by vhuseinova-msft — APPROVED** [review]

File: packages/react-components/src/components/styles/RichTextEditor.styles.ts (hunk 1 above)
Comment (palatter) [reply packages/react-components/src/components/styles/RichTextEditor.styles.ts:293]: Tested in both

**Comment by palatter** [comment]
> @palatter Do you know why the "table cells" doesn't look square on mobile? image

As discussed offline, we can create a seperate lower priority issue for this.

**Pull request merged**